src/ToolBox/superpmi/mcs/verbmerge.cpp

```diff
@@ -15,42 +15,52 @@
 // The caller must delete the returned string with delete[].
 //
 // static
-char* verbMerge::MergePathStrings(const char* dir, const char* file)
+LPWSTR verbMerge::MergePathStrings(LPCWSTR dir, LPCWSTR file)
 {
-    size_t dirlen  = strlen(dir);
-    size_t filelen = strlen(file);
+    size_t dirlen  = wcslen(dir);
+    size_t filelen = wcslen(file);
     size_t newlen  = dirlen + 1 /* slash */ + filelen + 1 /* null */;
-    char*  newpath = new char[newlen];
-    strcpy(newpath, dir);
-    strcat(newpath, DIRECTORY_SEPARATOR_STR_A);
-    strcat(newpath, file);
+    LPWSTR newpath = new WCHAR[newlen];
+    wcscpy(newpath, dir);
+    wcscat(newpath, DIRECTORY_SEPARATOR_STR_W);
+    wcscat(newpath, file);
     return newpath;
 }
 
+char* verbMerge::ConvertUTF8ToMultiByte(LPCWSTR wstr)
+{
```

Comment:
I don't understand this. UTF8 strings should be stored in `char*`, whereas LPCWSTR should store a string of 2-byte Unicode characters (or maybe UTF-16).